### ISSUE TITLE
[Enh] migration generation

### ISSUE THREAD

**Issue by Faryshta** [yiisoft/yii2-gii#151]
An interface to use the `migrate/create` console command would be great, specially to generate the fields.

**Comment by cebe** [comment]
there is already a tool like this: http://yiifeed.com/news?id=116

**Comment by XzAeRo** [comment]
Yeah, there are a few tools around for this purpose. But I believe it would be great to ship this feature bundled in Gii.
An option to generate migrations with seeded data would be awesome.

**Comment by Faryshta** [comment]
@cebe i checked it, sandstorm requires node and NPM, doesn't use the syntax defined in 2.0.7 and doesn't have foreign keys support.

I honestly prefer to use the command line so I would rather have a generator in gii what can emulate the command line behavior

**Comment by mklemarczyk** [comment]
Do you mean code first approach? To generate migration from changed model?
